app/Http/Controllers/Backend/PermissionsController.php

```diff
@@ -0,0 +1,253 @@
+<?php
+namespace App\Http\Controllers\Backend;
+
+use Illuminate\Http\Request;
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Models\Backend\Permission;
+
+class PermissionsController extends Controller
+{
+    /**
+     * Display a listing of the Permissions.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Permission::all();
+        return view('backend.permissions.index')->with('data', $data);
```

Comment:
[rank B] please check this again, you can use:
`return view('backend.permissions.index')->with($data);`